qiskit_ibm_runtime/__init__.py

```diff
@@ -218,6 +219,7 @@ def result_callback(job_id, result):
 from .estimator import Estimator
 from .sampler import Sampler
 from .options import Options
+from .runner_result import RunnerResult
```

Comment:
`RunnerResult` doesn't need to be in `__init__` since we don't expect users to call `circuit-runner` directly. It can be in the same directory as the other result decoders moved in #511, or a new one if `utils` get too crowded.